### PR TITLE
fix addressvalidation.py 

### PR DESCRIPTION
**Pull request by anglarett**
https://github.com/googlemaps/google-maps-services-python/pull/450

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,5 @@ googlemaps.egg-info
 *.egg
 .vscode/
 .idea/
+index.py
```
Comment (amuramoto) [inline .gitignore:37]: Does this need to be added to gitignore?

File: .gitignore [hunk 2]
```diff
@@ -34,3 +34,5 @@ googlemaps.egg-info
...
+test.py
```
Comment (amuramoto) [inline .gitignore:38]: Does this need to be added to gitignore?

File: googlemaps/addressvalidation.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    if locality is not None:
+        params["address"]["locality"] = locality
+
+    if enableUspsCass is not False or enableUspsCass is not None:
+        params["enableUspsCass"] = enableUspsCass
+
+    return client._request("/v1:validateAddress", {},  # No GET params
+                           base_url=_ADDRESSVALIDATION_BASE_URL,
+                           extract_body=_addressvalidation_extract,
+                           post_json=params)
```
Comment (amuramoto) [inline googlemaps/addressvalidation.py:80]: Add newline at EOF for linter

Comment (amuramoto) [inline googlemaps/addressvalidation.py:80]: Add newline at EOF for linter

File: googlemaps/addressvalidation.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
+#
+# Copyright 2014 Google Inc. All rights reserved.
```
Comment (amuramoto) [inline googlemaps/addressvalidation.py:2]: Change license header to 2022

Comment (anglarett) [reply googlemaps/addressvalidation.py]: I will add this change

Note: I will submit a new pull request as single commit for all changes to this review round

File: googlemaps/addressvalidation.py (hunk 3 above)
Comment (anglarett) [reply googlemaps/addressvalidation.py]: I will add this change

Note: I will submit a new pull request as single commit for all changes to this review round

File: .gitignore (hunk 2 above)
Comment (anglarett) [reply .gitignore]: I will remove test.py

Note: I will submit a new pull request as single commit for all changes to this review round

File: .gitignore (hunk 1 above)
Comment (anglarett) [reply .gitignore]: I need this for local development.

Note: I will submit a new pull request as single commit for all changes to this review round

**Review by amuramoto — APPROVED** [review]

**Pull request merged**